### ISSUE TITLE
In example Double click on modify features can lead to Uncaught TypeError

### ISSUE THREAD

**Issue by PeterC66** [openlayers/openlayers#7710]
In the latest (v4.6.4) modify features example (https://openlayers.org/en/latest/examples/modify-features.html) if I select a country, move a vertex, and then double click on it I quite often get an Uncaught TypeError. I don't know how to get ol-debug for the example but the stack trace I see starts ol.js:369 Uncaught TypeError: Cannot read property 'fa' of undefined. I attach a screen shot of an example. I use Chrome but get the same thing in Edge. ![issue02](https://user-images.githubusercontent.com/16805744/34974684-47378532-fa86-11e7-8c21-cf694b76308c.jpg)

The resulting behaviour seems to be that the map has a point where I double clicked, and a vertex point has followed my mouse, and the map is panning with the mouse and cannot be stopped from panning.
I noticed this in my own code first, and thought it strange, so I came to the example to see what ought to happen. 

There is no need to address my own code here, but (using v4.0.0/ol-debug.js) the stack trace I get when I do something similar is
ol-debug.js:27898 Uncaught TypeError: Cannot read property 'minX' of undefined
    at ol.structs.RBush.update (ol-debug.js:27898)
    at ol.interaction.Modify.handleUpEvent_ (ol-debug.js:65025)
    at ol.interaction.Modify.ol.interaction.Pointer.handleEvent (ol-debug.js:18306)
    at ol.interaction.Modify.handleEvent (ol-debug.js:65071)
    at ol.Map.handleMapBrowserEvent (ol-debug.js:31487)
    at ol.MapBrowserEventHandler.boundListener (ol-debug.js:6671)
    at ol.MapBrowserEventHandler.ol.events.EventTarget.dispatchEvent (ol-debug.js:7091)
    at ol.MapBrowserEventHandler.handlePointerUp_ (ol-debug.js:12358)
    at ol.pointer.PointerEventHandler.boundListener (ol-debug.js:6671)
    at ol.pointer.PointerEventHandler.ol.events.EventTarget.dispatchEvent (ol-debug.js:7091)

I have seen various issues raised around double clicking (eg #6310) but none with the actual OL example.

**Comment by AnthonNilsson** [comment]
I have noticed the same problem (same stacktrace). But have a slightly different way to get the error to show. (Maybe it is the same, but the above just seems to be this problem)

If using modify on selected features i.e.: 
https://openlayers.org/en/latest/examples/modify-test.html

1. Select a line.
2. double click, but do not release on the second click.
3. Now while keeping the mouse button pressed drag and edit the line.
4. Release the mouse. This gives for me the above error every time.

**Comment by OldGreyCells** [comment]
This TypeError appears to be triggered by a call to RBush.prototype.update (where 'item' is undefined) from the modify handleUpEvent function, line 1158: this.rBush_.update(boundingExtent(segmentData.segment), segmentData);.

**Comment by AnthonNilsson** [comment]
Hi,

The last example seems not working with this anymore, but this one does;
https://openlayers.org/en/latest/examples/modify-features.html
Follow the same steps as I wrote before and the error will come every time.

I also modified the code and tested select.setActive(false) when starting to modify. This do not work either. I think the problem is that the event 'modifystart' happens first when starting to drag the feature, and not when mouse is down.
https://stackblitz.com/edit/typescript-fa1fg8?file=index.ts

In my own application if testing at lines, and deactivating select, I was able to do the workaround once per line though. The 'modifystart' event was fired every mouse down time. But only once, after the first time a feature was modified the event fired only when dragging again, I have some more logic then only the basic ol one though, but do not think this should have any effect on this behavior.

This issue makes the production app crash frequently, so I would really appriciate if it could be checked at. And it seems that this issue exists for all select and modify applications.
